### ISSUE TITLE
Development and Support

### ISSUE THREAD

**Issue by alirezag** [mil-tokyo/webdnn#815]
Hi all,

I was wondering if WebDNN is officially or unofficially supported by Chainer/PFN? I see the development is more advanced for Chainer. For example, batchnormalization support is enabled for Chainer. 

We are team developing some AI toolkits are interested in using WebDNN but we are wondering one of the AI source frameworks have a better support on WebDNN.

Cheers,

**Comment by milhidaka** [comment]
WebDNN is not supported by PFN.
Chainer is the first supported framework simply because I mainly use Chainer for deep learning (as it is well known in Japan).
Currently we are not planning to support other new frameworks, but posting issues for any supported frameworks (Chainer, Caffe, Keras, Tensorflow, PyTorch) are welcomed.

**Comment by alirezag** [comment]
Thanks guys.
